main/com.spotify/index.js

```diff
@@ -1103,7 +1102,7 @@ module.exports = class SpotifyDevice extends Tp.BaseDevice {
         let album_uris = [];
         let album_tracks = {};
         try {
-            await this.do_player_shuffle("false");
+            await this.do_player_shuffle("false", env);
```

Comment:
Wait does this code mean that we call set shuffle once for every song now? That still seems bad.

And more generally, given we no longer expand albums to a list of songs, should we be setting shuffle at all?

Reply:
Oh that's a good point, I forgot we changed this.